topo_processor/cli/main_test.py

```diff
@@ -0,0 +1,33 @@
+import json
+import os
+import shutil
+from tempfile import gettempdir
+
+import pytest
+
+from topo_processor.metadata import DataType, create_collection
+
+from .main import upload_to_local_disk
+
+
+@pytest.mark.asyncio
+async def test_upload_to_local_disk():
+
+    source = os.path.join(os.getcwd(), "test_data", "tiffs", "C8054")
+    datatype = "imagery.historic"
+    target = gettempdir()
+
+    collection = await create_collection(source, DataType(datatype))
+    await upload_to_local_disk(collection, target)
+
+    assert os.path.isfile(os.path.join(target, "C8054", "29659.json"))
+    assert os.path.isfile(os.path.join(target, "C8054", "29659.tiff"))
+    assert os.path.isfile(os.path.join(target, "C8054", "collection.json"))
+
+    with open(os.path.join(target, "C8054", "29659.json")) as item_json_file:
+        item_metadata = json.load(item_json_file)
+    assert (
+        item_metadata["properties"]["checksum:multihash"]
+        == "1220f7ea9017ef596cc650369493dc3993d9ab5c99055bbc5c7bac9d4e061c5f2f7a"
+    )
+    shutil.rmtree(os.path.join(target, "C8054"))
```

Comment:
do more checking:
1. collection.json has pointers to items
2. item json has pointer to tiff

Reply:
look for a better way to handle tmp dir (e.g. with)